Extensions/auto_tagger.js

```diff
@@ -131,8 +131,20 @@ XKit.extensions.auto_tagger = new Object({
 			text: "Tag with date (ie: <i>#August 21th 2013, #August, #21th, #2013</i>)",
 			default: false,
 			value: false
-		}
+		},
 
+		"tag_source": {
+			text: "When reblogging, tag with the username of the original source",
```

Comment:
probably should add something like ", if possible", because posts where the source have been deleted won't show up.